Source/DafnyLanguageServer.Test/Lookup/HoverVerificationTest.cs

```diff
@@ -210,16 +214,20 @@ method Abs(x: int) returns (y: int)
   }
   assert {:split_here} true;
 } ", "testfileSlow.dfy", CompilationStatus.VerificationFailed);
-      await Task.Delay(100); // Just time for the diagnostics to be updated
-      await AssertHoverMatches(documentItem, (0, 22),
-        @"**Verification performance metrics for method SquareRoot2NotRational**:
+        await Task.Delay(100); // Just time for the diagnostics to be updated
```

Comment:
Off topic BUT: is it possible to replace the `await Task.Delay(100)` calls with `await GetLastDiagnostics(..)` ?